commit 5ceea8e508640f9044b08d15dd78407bb2bba9f3
Author: Valentin David <me@valentindavid.com>
Date:   Sat Jan 24 12:13:27 2026 +0100

    lvm2: Disable event activation by default
    
    Because lvm2 does not have a distro wide configuration in /usr and all
    is in /etc, configuring it is not compatible with systemd-sysupdate
    OSes - because systemd-udevd starts before systemd-tmpfiles. Also on
    GNOME OS, installation is confusing for the user that cannot format a
    disk before they deactivate the volume groups taking phyisical volumes
    from that disk.
    
    The solution is to disable this by default
    
    Now, Bluefin users will be fine with lvm2 installations, as
    bootc/OSTree still provides /etc at the right time. So for those
    distros we will set the initial value in /etc to activate the groups.
    
    It does not seem Endless OS uses lvm2 volume groups. Furthermore, it
    could also set the configuration in /etc since it uses OSTree.

diff --git a/elements/components/lvm2-base.bst b/elements/components/lvm2-base.bst
index 5f289a56a..5b24218eb 100644
--- a/elements/components/lvm2-base.bst
+++ b/elements/components/lvm2-base.bst
@@ -22,6 +22,7 @@ variables:
     --with-staticdir=%{sbindir}
     --with-usrlibdir=%{libdir}
     --with-usrsbindir=%{sbindir}
+    --with-default-event-activation=0
 
 config:
   configure-commands:
